tools/fast/action_show.go

```diff
@@ -23,11 +23,11 @@ func (f *Filecoin) ShowBlock(ctx context.Context, ref cid.Cid) (*types.Block, er
 }
 
 // ShowDeal runs the `show deal` command against the filecoin process
-func (f *Filecoin) ShowDeal(ctx context.Context, ref cid.Cid) (*storagedeal.Deal, error) {
+func (f *Filecoin) ShowDeal(ctx context.Context, dealResp *storagedeal.Response) (*storagedeal.Deal, error) {
```

Comment:
Do we want to pass in the entire `storagedeal.Response` here?

Reply:
I initially had it take the CID and that would match the other function. I'm fine with changing it back.